<?xml version="1.0"?>
<launch>

        <!--Nodo_ODOM -->
        
        <node pkg="tf2_ros" type="static_transform_publisher" name="map_joint" args="odomPose_x  odomPose_y  odomPose_z  odomPose_roll  odomPose_pitch  odomPose_yaw   map   odom"/>

        <!--Load mesh model -->
        <arg name="puzzlebot_sim_model" default="$(find puzzlebot_sim)/urdf/puzzlebot_jetson_ed_urdf.urdf"/>
        <param name="robot_description" command="cat $(arg puzzlebot_sim_model)" />

        <!--Pruebas -->
        <!--<node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui"/>
       --> <!--Robot state publisher??? -->
    
        <node pkg="robot_state_publisher" type="robot_state_publisher" name="simulated_state_publisher" output="screen"/>

        <!--Joint State Publisher -->     
        <node name="puzz_sim" pkg="puzzlebot_sim" type="puzzlebot_kinematic_model.py"/>

        <node name="localisation" pkg="puzzlebot_sim" type="localisation.py"/>

        <node name="controller" pkg="puzzlebot_sim" type="Control2.py"/>

    
    
    
        <!--RVIZ -->
        <arg name="rvizconfig" default="$(find puzzlebot_sim)/rviz/puzzlebot_sim.rviz" />
        <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" /> 

</launch>


